js/index.js

```diff
@@ -63,9 +63,11 @@ app.controller("IndexController", ["$scope", "$http", 'leafletData', function($s
 
   function loadTodos(response) {
     if (response.data.length > 0) {
+      var dateString;
       $scope.markers = response.data;
       $($scope.markers).each(function(index, marker) {
-        $scope.markers[index].message = "<b>Title: </b>" + marker.title + "<br/><b>Due Date: </b><b>" + marker.dueDate + "</b>";
+        dateString= new Date(marker.dueDate).toGMTString();
+        $scope.markers[index].message = "<b>Title: </b>" + marker.title + "<br/><b>Due Date: </b>" + dateString.substring(0, dateString.length - 4);
```

Comment:
Can you move the substring() operation to the previous line to make this line less long?